### ISSUE TITLE
Accessibility Issue: Empty Links and links with images

### ISSUE THREAD

**Issue by wakingmorning** [badcamp/badcamp-2018#164]
### Description
- Links that contain an image need either text, or alt text on the image, otherwise screen readers will read the URL.
- Links that have no image or text should have a `title`, `aria-label` attribute. 

### Issues
- The sponsor logos should have an alt tag on the images.
    <img width="1141" alt="screen shot 2018-08-21 at 3 58 51 pm" src="https://user-images.githubusercontent.com/6785398/44425869-2370c480-a55b-11e8-8864-bd52d681e062.png">
- The social links should have an `aria-label` or `title` attribute
    <img width="398" alt="screen shot 2018-08-21 at 3 59 30 pm" src="https://user-images.githubusercontent.com/6785398/44425920-40a59300-a55b-11e8-82b0-b6c9e867e21f.png">
